packages/bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -31,47 +32,106 @@ export type Props = {
   className: ?string,
   leadingScrollIndicatorClassName: ?string,
   trailingScrollIndicatorClassName: ?string,
+  autoScrollToSelected: boolean,
 };
 
-const BpkHorizontalNav = (props: Props) => {
-  const classNames = [getClassName('bpk-horizontal-nav')];
-  const {
-    children,
-    className,
-    leadingScrollIndicatorClassName,
-    trailingScrollIndicatorClassName,
-    ...rest
-  } = props;
-
-  // Outer classNames
-  if (className) {
-    classNames.push(className);
+const getLeft = (ref: any): number => {
+  if (!ref) {
+    return 0;
   }
-
-  return (
-    <BpkMobileScrollContainer
-      innerContainerTagName="nav"
-      className={classNames.join(' ')}
-      leadingIndicatorClassName={leadingScrollIndicatorClassName}
-      trailingIndicatorClassName={trailingScrollIndicatorClassName}
-      {...rest}
-    >
-      <ul className={getClassName('bpk-horizontal-nav__list')}>{children}</ul>
-    </BpkMobileScrollContainer>
-  );
+  const pos = ref.getBoundingClientRect();
+  return pos.left;
 };
 
-BpkHorizontalNav.propTypes = {
-  children: PropTypes.node.isRequired,
-  className: PropTypes.string,
-  leadingScrollIndicatorClassName: PropTypes.string,
-  trailingScrollIndicatorClassName: PropTypes.string,
-};
+class BpkHorizontalNav extends Component<Props> {
+  selectedItemRef: ?Element | ?React$Component<any, any>;
```

Comment:
`?React$Component<any, any>;`

Surely this should be React.Element? https://flow.org/en/docs/react/types/#toc-react-element